tvPresentation.js

```diff
@@ -0,0 +1,231 @@
+        screenType: 'LED',
+        screenQuality: 'Ultra HD/4K',
+        smartTv: true,
+        options: {
+            wifi: true,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: true,
+        },
+        originalStock: 6,
+        sold: 3,
+    },
+    {
+        type: 'B2450HD',
+        name: 'LED TV',
+        brand: 'Brandt',
+        price: 109,
+        availableSizes: [24],
+        refreshRate: 60,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: false,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+    {
+        type: '32WL1A63DG',
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
+    },
+];
+
+// const newStringArray = inventory.map(( newString ) => {
+//
+//     return inventory.brand + inventory.type + "-" + inventory.name;
+//
+// });
+//
+// function newString(){
+//     return newStringArray.join("");
+// }
+//
+// console.log(newStringArray);
+
+
+
+function brandAndType( input )    {
+    return inventory[input].brand + " " + inventory[input].type + " - " + inventory[input].name;
+}
+
+const tvBrandAndType = document.getElementById("brand-type");
+tvBrandAndType.textContent = brandAndType(3)
+
+// console.log(brandAndType( 1 ));
+
+function tvPrice( input )   {
+    const tvPriceFormat = inventory[input].price
+    return "€ " + Math.round(tvPriceFormat) + ",-";
+}
+
+const tvPriceSet = document.getElementById("price");
+tvPriceSet.textContent = tvPrice(3);
+
+// console.log(tvPrice(0));
+
+// function screenSizes( input )   {
+//     const screenSize = inventory[input].availableSizes
+//     for (let i = 0; i < inventory; i++) {
+//
+//     }
+// }
+// const outcome = inventory[0].availableSizes.map((screenSize) => {
+//
+//     return `Screen sizes are: ${screenSize} inches | In centimeters: ${screenSize * 2.54} cm`;
+//
+// })
+//
+// // console.log( outcome );
+//
+// const screenSizesTv = document.getElementById("tv-sizes");
+// screenSizesTv.textContent = outcome;
+
+
+function availableScreenSizes ( input ){
+    let size = "";
+
+    for (let i = 0; i < input.length; i++) {
+        const sizeInInch = input[i];
+        const sizeInCM = input[i] * 2.54;
+        //const stringScreenSize = `Screen sizes are: ${sizeInInch} inch | In centimeters that is: ${sizeInCM} cm`;
+        size = size + sizeInInch + ` inch | In centimeters that is: ${sizeInCM} cm \n`
```

Comment:
Hier zou je nog dit kunnen doen:
```suggestion
        size += sizeInInch + ` inch | In centimeters that is: ${sizeInCM} cm \n`
```